notfications/report.py

```diff
@@ -125,8 +125,8 @@ def publish_list_to_topic(sns_client: client,
         item_type = item['type'].title()
         item_url = item['url']
 
-        message += f"{item_type} '{item_name}' by '{
-            item_author}' is currently trending!\n"
+        message += f'''{item_type} '{item_name}' by '{
+            item_author}' is currently trending!\n'''
```

Comment:
" > ' imo

Reply:
The lambda image doesn't recognise " for multiple lines which is why I changed it to ' instead.